load-test-framework/src/main/java/com/google/pubsub/flic/controllers/GCEController.java

```diff
@@ -431,4 +449,28 @@ private InstanceTemplate defaultInstanceTemplate(String type) {
             .setServiceAccounts(Collections.singletonList(new ServiceAccount().setScopes(
                 Collections.singletonList("https://www.googleapis.com/auth/cloud-platform")))));
   }
+
+  /**
```

Comment:
I moved the code to the sheets service for now. This works for now, but when we get more clients, I'll add the changes we discussed to read client type, check cps/kakfa and publish/subscribe, before adding to the spreadsheet.